### ISSUE TITLE
Warning about enhanced colors when using linux tty[1-6] console

### ISSUE THREAD

**Issue by edrozenberg** [richrd/suplemon#253]
When using the local tty (1-6), the default terminal is set to "linux". This default terminal only supports 8 colors ( `tput colors` shows 8 ). suplemon prints the following warning at each program start as a result:

```
suplemon.ui - WARNING - Enhanced colors failed to load. You could try 'export TERM=xterm-256color'.
```

Suggestion: don't print warning, or provide a setting to ignore this issue and not print a warning each time.

**Comment by richrd** [comment]
Yeah, it's probably reasonable to not show that as a warning. Maybe it could be set to INFO or even DEBUG. I'll probably change it in the next release.
